src/modules/launcher/Plugins/Microsoft.Plugin.Uri.UnitTests/UriHelper/ExtendedUriParserTests.cs

```diff
@@ -11,87 +11,161 @@ namespace Microsoft.Plugin.Uri.UnitTests.UriHelper
+        [DataRow("https://127.0.0.1/test", true, "https://127.0.0.1/test", true, null)]
+        [DataRow("https://127.0.0.1:1234/test", true, "https://127.0.0.1:1234/test", true, null)]
+        [DataRow("[::1]", true, "https://[::1]/", true, null)]
+        [DataRow("[::1]:80", true, "http://[::1]/", true, null)]
+        [DataRow("[::1]:443", true, "https://[::1]/", true, null)]
+        [DataRow("[::1]:1234", true, "https://[::1]:1234/", true, null)]
+        [DataRow("[::1]/test", true, "https://[::1]/test", true, null)]
+        [DataRow("[::1]:80/test", true, "http://[::1]/test", true, null)]
+        [DataRow("[::1]:443/test", true, "https://[::1]/test", true, null)]
+        [DataRow("[::1]:1234/test", true, "https://[::1]:1234/test", true, null)]
+        [DataRow("http://[::1]", true, "http://[::1]/", true, null)]
+        [DataRow("http://[::1]:1234", true, "http://[::1]:1234/", true, null)]
+        [DataRow("https://[::1]", true, "https://[::1]/", true, null)]
+        [DataRow("https://[::1]:1234", true, "https://[::1]:1234/", true, null)]
+        [DataRow("http://[::1]/test", true, "http://[::1]/test", true, null)]
+        [DataRow("http://[::1]:1234/test", true, "http://[::1]:1234/test", true, null)]
+        [DataRow("https://[::1]/test", true, "https://[::1]/test", true, null)]
+        [DataRow("https://[::1]:1234/test", true, "https://[::1]:1234/test", true, null)]
+
+        // Case where `domain:port`, as specified in issue #14260
+        // Assumption: Only domain with dot is accepted
+        [DataRow("example.com:80", true, "http://example.com/", true, null)]
+        [DataRow("example.com:80/test", true, "http://example.com/test", true, null)]
+        [DataRow("example.com:80/126", true, "http://example.com/126", true, null)]
+        [DataRow("example.com:443", true, "https://example.com/", true, null)]
+        [DataRow("example.com:443/test", true, "https://example.com/test", true, null)]
+        [DataRow("example.com:443/126", true, "https://example.com/126", true, null)]
+        [DataRow("google.com:91", true, "https://google.com:91/", true, null)]
+        [DataRow("google.com:91/test", true, "https://google.com:91/test", true, null)]
+        [DataRow("google.com:91/126", true, "https://google.com:91/126", true, null)]
+        [DataRow("test.test.test.test:952", true, "https://test.test.test.test:952/", true, null)]
+        [DataRow("test.test.test.test:952/test", true, "https://test.test.test.test:952/test", true, null)]
+        [DataRow("test.test.test.test:952/126", true, "https://test.test.test.test:952/126", true, null)]
+
+        // Following cases can be both interpreted as schema:path and domain:port
+        [DataRow("tel:411", true, "https://tel:411/", true, "tel:411")]
+        [DataRow("tel:411/test", true, "https://tel:411/test", true, "tel:411/test")]
+        [DataRow("tel:411/126", true, "https://tel:411/126", true, "tel:411/126")]
```

Comment:
which use case should this be? I don't know telephone numbers with a slash and text.

Reply:
Hmm, should I remove it or make it a negative test case? It's added because the regex filters for numbers and there was concern earlier it'd break with /

If we want to filter tel protocol with / it will have to be another filter, but I'm fine with that also

Reply:
let's remove these tests. I don't think that someone will input something like this for a telephone number. So, no need for filtering this input.